app/models/openscap_result.rb

```diff
@@ -0,0 +1,76 @@
+class OpenscapResult < ApplicationRecord
+  belongs_to :container_image
+  has_one    :binary_blob,           :dependent => :destroy, :autosave => true, :as => :resource
+  has_many   :openscap_rule_results, :dependent => :destroy, :autosave => true
+
+  def self.openscap_available?
```

Comment:
How about moving the openscap requirement into its own class?
By default you just delegate the methods and in a testing env you give it the stub.
This way you could also split the tests into a) testing OpenscapResult and b) testing the correct use of the openscap gem
Especially b) is a problem with mocks, as @simon3z pointed out

I just recently came a long a good blog post about this:
http://blog.plataformatec.com.br/2016/02/experimenting-with-explicit-contracts-with-ruby/